lib/ansible/playbook/play.py

```diff
@@ -64,6 +64,9 @@ class Play(Base, Taggable, Become):
 
     # Connection
     _gather_facts        = FieldAttribute(isa='bool', default=None, always_post_validate=True)
+    _gather_subset       = FieldAttribute(isa='string', default=None, always_post_validate=True)
```

Comment:
we might want to make this a list, so both an actual yaml list and , separated string will work.